src/Page/Food/Builder.elm

```diff
@@ -295,88 +302,83 @@ findExistingBookmarkName { builderDb, store } query =
+
+
+type alias UpdateProcessConfig =
+    { processes : List Process
+    , excluded : List Process.Code
+    , processQuery : Query.ProcessQuery
+    , impact : Html Msg
+    , updateEvent : Query.ProcessQuery -> Msg
+    , deleteEvent : Msg
+    }
+
+
+updateProcessFormView : UpdateProcessConfig -> Html Msg
+updateProcessFormView { processes, excluded, processQuery, impact, updateEvent, deleteEvent } =
+    li [ class "IngredientFormWrapper" ]
+        [ span [ class "MassInputWrapper" ]
+            [ MassInput.view
+                { mass = processQuery.mass
                 , onChange =
                     \maybeMass ->
-                        select
-                            (case ( maybeMass, selectedProcess ) of
-                                ( Just mass, Just selected ) ->
-                                    Just { selected | mass = mass }
-
-                                _ ->
-                                    Nothing
-                            )
-                , disabled = selectedProcess == Nothing
-                }
-            )
-            (db.processes
-                |> Process.listByCategory category
-                |> List.sortBy Process.getDisplayName
-                |> List.filter (\{ code } -> not (List.member code excluded))
-                |> processSelectorView kind
-                    (Maybe.map .code selectedProcess)
-                    (\maybeCode ->
-                        case ( selectedProcess, maybeCode ) of
-                            ( Just selected, Just code ) ->
-                                select (Just { selected | code = code })
-
-                            ( Nothing, Just code ) ->
-                                select (Just { code = code, mass = defaultMass })
+                        case maybeMass of
+                            Just mass ->
+                                updateEvent { processQuery | mass = mass }
 
                             _ ->
-                                select Nothing
-                    )
-            )
-            (button
-                [ type_ "submit"
-                , class "btn btn-sm btn-primary"
-                , title <| "Ajouter " ++ kind
-                , disabled <| selectedProcess == Nothing
-                ]
-                [ Icon.plus ]
-            )
+                                NoOp
+                , disabled = False
+                }
+            ]
+        , processes
+            |> List.sortBy (.name >> Process.nameToString)
+            |> processSelectorView
+                processQuery.code
+                (\code -> updateEvent { processQuery | code = code })
+                excluded
+        , span [ class "text-end ImpactDisplay" ]
+            [ impact ]
+        , button
+            [ type_ "button"
+            , class "btn btn-sm btn-outline-primary IngredientDelete"
+            , title <| "Supprimer "
+            , onClick deleteEvent
+            ]
+            [ Icon.trash ]
```

Comment:
For some reason I couldn't identify browsing the patch, ingredient delete buttons don't work anymore

Reply:
Bah, it was a weird Parcel cache issue on my temporary linux machine, solved by itself after clearing it.